lib/seed_migration/migration.rb

```diff
@@ -6,6 +6,20 @@ module SeedMigration
   class Migration
     include Environments
 
+    class << self
+      attr_accessor :disable_ddl_transaction
+      # Disable the transaction wrapping this migration.
+      # You can still create your own transactions even after
+      # calling #disable_ddl_transaction!
+      def disable_ddl_transaction!
+        @disable_ddl_transaction = true
+      end
+    end
+
+    def disable_ddl_transaction
+      self.class.disable_ddl_transaction
```

Comment:
Missing the `!` suffix on the method call here.

Reply:
Supposed to. Just an attr accessor.  The bang is to set the var